Feat/replace tooltip with popover

a-crea opened the pull request:
With this PR, we replace the TooltipCustom component with the PopoverCustom component.

gappc commented:
Hi @a-crea, the PR seems fine, I encountered just some minor issues during code review:

1. popover x-positioning seems wrong due to changes in [useFloatingUi.ts](https://github.com/noi-techpark/it.bz.opendatahub.databrowser/blob/b275c35c93faccae63341fb4deedae6f2f2f2996/databrowser/src/components/utils/useFloatingUi.ts), see image below. What were the reasons for the changes?
2. popover arrow seems to be a rectangle, not an arrow (see image below)
3. Code formatting (see comment)
4. not sure if method `onPopoverButtonClick` is needed, please check (see comment)

![image](https://github.com/user-attachments/assets/ad8ff5dc-965e-480f-ae88-5ac1e75ea15f)

a-crea commented:
hi @gappc :)
Thanks for the feedback!
I’ve edited useFloatingUi because, in the table view, the arrow wasn’t aligning correctly with the popover panel, causing a situation similar to what you showed in your screenshot. I adjusted the strategy and cleaned up the code. It should be fine now—please let me know! :)

RudiThoeni commented:
Please resolve merge conflicts and let me know if this PR is ready for merging?

gappc commented:
@RudiThoeni @a-crea sorry, my fault I forgot to take a look at it. @a-crea would it be possible that you fix the conflicts?

gappc commented:
@a-crea the PR looks fine to me, thank you for your help :+1: 

@RudiThoeni this PR is ready for merge.
